src/main/java/com/aws/greengrass/clientdevices/auth/iot/Certificate.java

```diff
@@ -79,10 +85,10 @@ public void setStatus(Status status, Instant lastUpdated) {
 
     /**
      * Check certificate status.
-     * @return true if this certificate is active in IoT Core.
+     * @return true if this certificate is verified and active in IoT Core.
      */
     public boolean isActive() {
-        return status == Status.ACTIVE;
+        return status == Status.ACTIVE && isStatusTrusted();
```

Comment:
Based on the method name, I would assume this is essentially just shorthand for `cert.getStatus() == Status.ACTIVE`. That's not the case though. We should consider overriding `getStatus` as well

Reply:
should we have a method something like `getTrustedStatus()` ?

Reply:
overidden `getStatus()` to return the trusted status.